#160451586 Add and configure travis.yml for CI

andela-phemmy opened the pull request:
#### What does this PR do?
  Add and configure travis.yml for continous integration
#### Description of Task to be completed?
  1. Add travis.yml file
  2. configured travis to include node in build system
  3. updated readme to include travis badge
#### Any background context you want to provide?
   Continuous integration is needed in other to check the integrity of a system remotely.
#### What are the relevant pivotal tracker stories?
   [Pivotal tracker story](https://www.pivotaltracker.com/story/show/160451586 "Pivotal Tracker Board")

Commit:
chore(eslint): integrate eslint

Integrate Eslint to the project
to make sure the team is following coding conventions

closes 160451588

Commit:
chore(houndci): integrate houndci

Integrate HoundCI to the project
to also make sure the team is following coding conventions

closes 160451589

Commit:
chore(CI): add travis.yml file for continous integration

Commit:
chore(CI): add travis.yml file for continous integration

koiic reviewed (approved):
Looks good to me

Victor-Ugwueze reviewed (approved):
LGTM

KaiserPhemi commented on README.md:
```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/andela/lotus-ah.svg?branch=develop)](https://travis-ci.org/andela/lotus-ah)
 Authors Haven - A Social platform for the creative at heart.
```
#### Nitpick
This should be on a new line:
<img width="965" alt="screen shot 2018-09-25 at 2 18 32 pm" src="https://user-images.githubusercontent.com/26272628/46017062-7e538980-c0ce-11e8-8fd6-8c7b03f49938.png">

andela-phemmy replied:
@KaiserPhemi, I have implemented the feedback and pushed

KaiserPhemi commented on .travis.yml:
```diff
@@ -0,0 +1,10 @@
+  - postgresql
+
+before_script:
+  - psql -c 'create database travis_ci_test;' -U postgres
```
#### Nitpick
New line after the last line. For context check [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

KaiserPhemi reviewed (changes requested):
#### Nitpick
Implement feedback.

Commit:
chore(eslint): integrate eslint

Integrate Eslint to the project
to make sure the team is following coding conventions (added new line)

closes 160451588

Commit:
fix(CI): added new line to travis.yml and README.md

Commit:
fix(CI): added new line to travis.yml and README.md

Commit:
chore(houndci): integrate houndci

Integrate HoundCI to the project
to also make sure the team is following coding conventions (ammended)

closes 160451589

KaiserPhemi committed:
Merge pull request #4 from andela/ch-integrate-houndci-160451589

Ch integrate houndci 160451589

KaiserPhemi commented:
#### Critical
@andela-phemmy 
Please fix the merge conflicts

Commit:
fix(CI): fix conflicts between houndci and travisci badges

andela-phemmy commented:
@KaiserPhemi, I have fixed the merge conflicts
